admin_manual/configuration_server/activity_configuration.rst

```diff
@@ -11,7 +11,7 @@ to your users for various events like:
 * A file or folder has been deleted
 
 Users can see actions (delete, add, modify) that happen to files they have access to.
-Sharing actions are only visible to the sharer and sharee.
+Sharing actions are only visible to the sharer and share.
```

Comment:
This one was correct. `sharee` refers to the person who received the share. cc @nickvergessen

Reply:
oops. sorry, I must have misread that. I'll remove that from the PR now.